core/src/main/java/hudson/model/Queue.java

```diff
@@ -2461,8 +2461,8 @@ public int compareTo(WaitingItem that) {
         }
 
         public CauseOfBlockage getCauseOfBlockage() {
-            long diff = timestamp.getTimeInMillis() - System.currentTimeMillis();
-            if (diff > 0)
+            long diff = System.currentTimeMillis() - timestamp.getTimeInMillis();
```

Comment:
This inverts the sign, making any queue items with positive queue delay show a negative prefixed number.